Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
Maybe we should create one project per service instead of one big project shared between all teams. @do3-2021/kudo-agent-team, @do3-2021/kudo-cli-team, @do3-2021/kudo-controller-team, @do3-2021/kudo-scheduler-team

Reply:
This is a good point. We need to discuss it with everybody on the project. Although we can merge this PR and change directory layout later ?

Reply:
There may be quite a few changes once the multi-service organization is defined. I think we need to talk about this BEFORE we merge